app/templates/identity.hbs

```diff
@@ -1,99 +1,140 @@
 {{page-title 'Identity'}}
 
-<h1 class='identity-heading'>
-  Identity Service Verification
-</h1>
-<div class='identity-container'>
-  {{#if (eq @model.profileStatus 'PENDING')}}
-    <p>Verification process is Pending!</p>
-    <button
-      type='button'
-      {{on 'click' (fn this.handleRefresh)}}
-    >Refresh</button>
-  {{else if (eq @model.profileStatus 'VERIFIED')}}
-    <p>Verification Done!</p>
-  {{else}}
-    {{#if (eq @model.profileStatus 'BLOCKED')}}
-      <p>Your previous chaincode is blocked for some reason. Please generate a
-        new chaincode.</p>
+{{#if this.isDev}}
+  <div class='identity-page'>
+    <img alt="header" class='identity-header' src='identity/header.svg' width='100%' />
```

Comment:
why we define width here

Reply:
That's a conflicting topic I am reading about. I think we can put it in CSS.